tests/data/mxnet_mnist/mnist_neo.py

```diff
@@ -112,11 +110,10 @@ def neo_preprocess(payload, content_type):
     if content_type != "application/vnd+python.numpy+binary":
         raise RuntimeError("Content type must be application/vnd+python.numpy+binary")
 
-    f = io.BytesIO(payload)
-    return np.load(f)
+    return np.asarray(json.loads(payload.decode("utf-8")))
 
 
-### NOTE: this function cannot use MXNet
+# NOTE: this function cannot use MXNet
```

Comment:
[INFO] For prod code, I'd probably also modify this to actually print something out, because that's much more discoverable than the comment. Not worth the effort for a test script, though.